lib/consult.rb

```diff
@@ -67,7 +67,7 @@ def env
 
     # Return only the templates that are relevant for the current environment
     def active_templates
-      templates.select(&:should_render?)
+      @templates.select(&:should_render?)
```

Comment:
Why this change? There's an `attr_reader` on `@templates`.

Reply:
True. I still disagree, but I'll remove this commit from from this branch.